### PR TITLE
Issue template and codeowners

### PR DESCRIPTION
**Pull request by gregnazario**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# **Please** keep this file ordered alphabetically by directory paths.
+
+# Owners for the `.github` directory and all its subdirectories.
+
+/.github/ @aptos-labs/developer-platform @aptos-labs/security
```
Comment (0xaptosj) [inline CODEOWNERS:5]: @gregnazario

Comment (gregnazario) [reply CODEOWNERS:5]: This was requested earlier by the security folks (you can see it in all the other repos).

Or are you saying I should be there too :)

Comment (0xaptosj) [reply CODEOWNERS:5]: no, github says `This CODEOWNERS file contains errors`

Comment (gregnazario) [reply CODEOWNERS:5]: Added the team, now there's no error.

**Review by 0xaptosj — APPROVED** [review]

**Pull request merged**